src/mmw/js/src/modeling/views.js

```diff
@@ -209,10 +210,15 @@ var ProjectMenuView = Marionette.ItemView.extend({
     saveProjectOrLoginUser: function() {
         if (App.user.get('guest')) {
             var self = this;
-            App.getUserOrShowLogin(function() {
-                self.model.setUserIdOnProjectAndScenarios();
-                self.model.saveInitial();
-            });
+
+            new modalViews.CreateAccountPersuasionModalView({
```

Comment:
Oh, good point! It's pretty trivial to wrap in a `App.user.fetch().always(function() { ...`, so I think I'll add that.